src/init/RuntimeConfig.ts

```diff
@@ -14,12 +16,12 @@ export class RuntimeConfig implements RuntimeConfigData {
 
   public reset(data: RuntimeConfigData): void {
     this.pport = data.port ?? 3000;
-    this.pbase = data.base ?? `http://localhost:${this.port}/`;
+    this.pbase = ensureTrailingSlash(data.base ?? `http://localhost:${this.port}/`);
     this.prootFilepath = data.rootFilepath ?? process.cwd();
   }
 
-  public get base(): string {
-    return this.pbase;
+  public getBase(trailingSlash = true): string {
+    return trailingSlash ? this.pbase : trimTrailingSlashes(this.pbase);
```

Comment:
As mentioned in offline discussion, I don't think we need the `trimTrailingSlashes` here. It would probably be better if that part just got removed from the FileResourceStore (which is going to happen soon(tm) so I would just keep that call there for now until it's changed).